scss-partials/_service.scss

```diff
@@ -0,0 +1,47 @@
+.service {
+    align-items: center;
+    text-align: center;
+    margin: 100px 0px 48px;
+
+    @include media ( "<=sm" ) {
+        margin: 64px 0px 0px;
+    }
+    
+    &__header {
+        color: $tws_black;
+        font-family: $font-heading;
+        font-style: normal;
+        font-size: 24px;
+        font-weight: 300;
+        line-height: 32px;
+        margin: 0px 16px;
+
+        @include media ( ">=md" ) {
+            font-size: 32px;
+            line-height: 40px;
+        }
+    }
+
+    &__info {
+        
+        &-text {
+            color: $tws_black;
+            font-family: $font-regular;
+            font-size: 16px;
+            line-height: 24px;
+            font-weight: 400;
+            max-width: 343px;
+            margin: 0 auto;
+            margin-top: 16px;
+            
+            @include media ( ">=md" ) {
+                max-width: 600px;
+                // previously had this as 588px as per the figma design but on live the 'journey' in the final sentence is on the third line not under it - where should this be?
+            }
```

Comment:
Here is where I added the max-width as 600px for medium screens upward. Shall I still change this to have the 16px padding and remove the inital 343px max width?

Reply:
you can remove the 343px max width and simply set 600 instead to all screens (you don't need media query at all). The section should keep horizontal padding, then you can adjust the width of the section given the design and what padding you choose.